packages/core/src/sass/app/modules/page-404.scss

```diff
@@ -0,0 +1,13 @@
+/** @define page-404 */
+.page-404 {
+    &__image {
+        background-image: url('~Images/common/404.png');
```

Comment:
what's the reason for background-image instead of a `<img>`?

Reply:
@oskar-binary I tried to find another place using .png and it was as a background image, so I followed the same. but I agree, <img> makes more sense. I'll update it.